spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1024,7 +1024,27 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       }
     }
   }
+  test("unhex") {
+    assume(!isSpark32, "unhex function has incorrect behavior in 3.2")
 
+    val table = "unhex_table"
+    withTable(table) {
+      sql(s"create table $table(col string) using parquet")
+
+      sql(s"""INSERT INTO $table VALUES
+        |('537061726B2053514C'),
+        |('737472696E67'),
+        |('\\0'),
```

Comment:
Removing this test fixes the Scala 2.13 build errors. I confirmed locally with `mvn clean package -DskipTests -Pscala-2.13`.

```suggestion
```